articles/synapse-analytics/sql-data-warehouse/design-guidance-for-replicated-tables.md

```diff
@@ -163,14 +163,15 @@ To ensure consistent query execution times, consider forcing the build of the re
 This query uses the [sys.pdw_replicated_table_cache_state](/sql/relational-databases/system-catalog-views/sys-pdw-replicated-table-cache-state-transact-sql?toc=/azure/synapse-analytics/sql-data-warehouse/toc.json&bc=/azure/synapse-analytics/sql-data-warehouse/breadcrumb/toc.json&view=azure-sqldw-latest&preserve-view=true) DMV to list the replicated tables that have been modified, but not rebuilt.
 
 ```sql
-SELECT [ReplicatedTable] = t.[name]
-  FROM sys.tables t  
-  JOIN sys.pdw_replicated_table_cache_state c  
-    ON c.object_id = t.object_id
-  JOIN sys.pdw_table_distribution_properties p
-    ON p.object_id = t.object_id
-  WHERE c.[state] = 'NotReady'
-    AND p.[distribution_policy_desc] = 'REPLICATE'
+SELECT SCHEMA_NAME(t.schema_id) SchemaName , [ReplicatedTable] = t.[name] , 
+'SELECT TOP 1 * FROM ' + '[' + SCHEMA_NAME(t.schema_id) + '].[' + t.[name] +']' AS RebuildStatement 
```

Comment:
```suggestion
SELECT SchemaName = SCHEMA_NAME(t.schema_id)
 , [ReplicatedTable] = t.[name]
 , [RebuildStatement] = 'SELECT TOP 1 * FROM ' + '[' + SCHEMA_NAME(t.schema_id) + '].[' + t.[name] +']'
```

Great edit @DiyaAlmuzaffar! Updated the TSQL for syntax consistency.

Reply:
Please also Edit the file again (in the files changed tab, or externally) to add a commit to update the `ms.date: 11/02/2021` to today's date.